src/audio/up_down_mixer/up_down_mixer.c

```diff
@@ -458,9 +437,7 @@ up_down_mixer_process(struct processing_module *mod,
 
 static const struct module_interface up_down_mixer_interface = {
 	.init = up_down_mixer_init,
-	.prepare = up_down_mixer_prepare,
```

Comment:
can we add comments to interface.h reset/prepare/free and mark these 3 are optional?